pyproject.toml

```diff
@@ -38,6 +39,9 @@ docs = [
     "sphinx-toolbox~=3.5",
     "sphinx-lint==1.0.0",
 ]
+apt = [
+    "python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu2/python-apt_2.4.0ubuntu2.tar.xz ; sys_platform == 'linux'"
+]
```

Comment:
```suggestion
```

This isn't something we want in all packages (we only need it for things that use python-apt), and IMO this isn't the way we'd want to do it anyway.